src/components/Select/types.ts

```diff
@@ -3,17 +3,20 @@ import {TextInputView, TextInputSize, TextInputPin} from '../TextInput';
 import {ControlGroupProps, ControlGroupOption, QAProps} from '../types';
 import {Option, OptionGroup} from './tech-components';
 
+export type RenderControl = (props: {
```

Comment:
This type is located in `index.ts`, I think it should have more unique name, maybe `SelectRenderControl`